NASA-AMMOS/aerie-ui
# Add toggle to autosize activity table columns

Resolves #346 

To test:

1. Create a plan
2. Add some directives and run a successful simulation
3. Verify that resizing the window or panel will cause the activity table columns to auto fit to the available space
4. Resize a column
5. Verify that resizing no longer auto fits the columns
6. Click on the auto size button to toggle it back on
![Screenshot 2023-11-29 at 2 55 38 PM](https://github.com/NASA-AMMOS/aerie-ui/assets/5290214/8eb3c23a-977a-43d7-9c1f-71c2fa605cb1)
7. Verify that resizing the window once again auto resizes the columns
